app/routes/_libraries.start.$version.index.tsx

```diff
@@ -172,7 +172,7 @@ export default function VersionIndex() {
           provider.
         </p>
         <Link
-          to="./docs/framework/react/start/overview"
+          to="../../router/latest/docs/framework/react/start/getting-started"
```

Comment:
Commit suggestion for the overview page if you want that instead. For your convenience 😆 
```suggestion
          to="../../router/latest/docs/framework/react/start/overview"
```